repos.d/rpm/openmandriva.yaml

```diff
@@ -0,0 +1,42 @@
+###########################################################################
+# OpenMandriva
+###########################################################################
+- name: openmandriva_3
+  type: repository
+  desc: OpenMandriva 3.x
+  family: openmandriva
```

Comment:
Please add this to `families` list in `repology-schemacheck.py`, this should also fix travis build